### ISSUE TITLE
Active?

### ISSUE THREAD

**Issue by jmrichardson** [ClimbsRocks/auto_ml#424]
Hi, Just wondering if this project is still active?  There hasn't been any commits in a while.  Thanks

**Comment by ClimbsRocks** [comment]
No, it's not really. I'd hoped it would be, but I've been too excited by other projects, and there are now several other production-grade automl services out there, unlike when I initially built this. I think I'm gonna keep expanding my open source time towards the fringes again, and try to make image recognition with massively pretrained models available to everyone, not just those with giant GPUs.

**Comment by jmrichardson** [comment]
This is unfortunate.  I have been looking around for open / python / free based AutoML tools that can utilize GPUs. This project looks really nice and wish you would reconsider.  However, I completely understand your motivation.  Could you recommend an alternative?

**Comment by ethomasii** [comment]
Azure AutoML is very similar to this, u may want to check it out.

**Comment by rolo** [comment]
h2o is really nice. https://github.com/h2oai/h2o-3

I've also heard good things about Ludwig but haven't had a chance to try it yet. https://github.com/uber/ludwig
